subprojects/packagefiles/flatbuffers/meson.build

```diff
@@ -1,32 +1,41 @@
 project(
     'flatbuffers',
     'cpp',
-    version : '1.11.0',
+    version : '2.0.0',
     license : 'Apache-2.0'
 )
 
 cpp = meson.get_compiler('cpp')
 
-cpp_args = [
-    '-Wall',
-    '-pedantic',
-    '-Werror',
-    '-Wextra'
-]
+cpp_args = []
+
+if host_machine.system() == 'windows'
+    cpp_args += '-D_CRT_SECURE_NO_WARNINGS'
+endif
+
+if cpp.get_argument_syntax() == 'gcc'
+    cpp_args += [
+        '-Wall',
+        '-pedantic',
+        '-Werror',
+        '-Wextra'
```

Comment:
This is what the meson builtin options `--werror` and `--warning-level 3` are for.

Reply:
And now I noticed that all you did was move this, it was there before. It's still wrong. :p